HarryPotter/Screens/Favourite/FavouriteViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  FavouriteViewController.swift
+//  HarryPotter
+//
+//  Created by Adrianna Mikulska on 28/09/2022.
+//
+
+import UIKit
+
+final class FavouriteViewController: UITableViewController {
+    
+    //MARK: - Properties
+    
+    private let cellId = "cellId"
+    private var favouriteViewModel: FavouriteViewModelProtocol
+    private var characters = [Character]() {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+    
+    //MARK: - Lifecycle
+    
+    init(favouriteViewModel: FavouriteViewModelProtocol) {
+        self.favouriteViewModel = favouriteViewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableRegister()
+        tableView.backgroundView = favouriteViewModel.backgroundView
+//        if let data = UserDefaults.standard.object(forKey: UserDefaultsKeys.jobCategory.rawValue) as? Data,
+//           let category = try? JSONDecoder().decode(JobCategory.self, from: data) {
+//             print(category.name)
+//        }
+        if let favouriteCharacter = UserDefaults.standard.object(forKey: favouriteViewModel.userKey) as? Data, // as [Characters]
+        let category = try? JSONDecoder().decode(Character.self, from: favouriteCharacter) {
+            print(category.name)
+        }
+    }
+    
+    //MARK: - Private
+    
+    private func tableRegister() {
+        tableView.register(FavouriteMemberCell.self, forCellReuseIdentifier: cellId)
+    }
+    private func downloadFav() {
+
+    }
```

Comment:
this is an empty function, can be removed? Or if it's needed in the future, you can add some comment, e.g. "//TODO: Make the API call to get favourites" or "//TODO: Fetch the favourites from UserDefaults"